amr-wind/wind_energy/ABLWallFunction.cpp

```diff
@@ -266,12 +267,19 @@ void ABLWallFunction::computeusingheatflux()
 
 ABLVelWallFunc::ABLVelWallFunc(Field&, const ABLWallFunction& wall_func)
     : m_wall_func(wall_func)
-{}
+{
+    amrex::ParmParse pp("ShearStress");
```

Comment:
I would prefer not adding another namespace here. Instead use `ABL.wall_shear_stress_type = moeng` ?